### DISCUSSION TITLE
How do I view a lemmy group in Misskey?

### DISCUSSION THREAD

**Discussion by creeperkafasi** [misskey-dev/misskey#11223] [Q&A] [answered]
I'm not sure if this is a limitation of Misskey or I am doing something wrong but when I can't seem to be able to open any lemmy groups in Misskey.
For example if I want to view https://lemmy.ml/c/linux on Mastodon I can; https://mastodon.social/@linux@lemmy.ml .
But when do the same on a Misskey instance https://misskey.io/@linux@lemmy.ml I get a page with a different image and no posts.
Is there any way to see these accounts in Misskey?

**Comment by kotking** [comment, marked as answer]
Groups implementation is still sketchy, if you look at mastodon one you will see that all messages are boosted by Group. Kbin for example does not self boost in magazines so it will be empty as you listed on Misskey. My work around if you need only one forum is to enable RSS reader and add your forum in RSS feed. Then use Lookup on Misskey any url from Lemmy forum and if it goes then you can view/reply from Misskey. Here I made a note showing process. https://misskey.io/notes/9hqdnhw749 . You can also just view on Lemmy and put link in Misskey to make an reply, that's how I use mine Mastodon.
